alkira/resource_alkira_service_fortinet_helper.go

```diff
@@ -1,18 +1,26 @@
 package alkira
 
 import (
+	"errors"
+	"fmt"
 	"log"
+	"os"
 
 	"github.com/alkiranet/alkira-client-go/alkira"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-func expandFortinetInstances(in []interface{}) []alkira.FortinetInstance {
+func expandFortinetInstances(licenseType string, in []interface{}, m interface{}) ([]alkira.FortinetInstance, error) {
```

Comment:
hmmm, why licenseType is a special one passed in? This should be part of the config... nope?

Reply:
LicenseType is required by the API to be passed in when creating the fortinet instance credentials and fortinet service.